src/components/CardItem.jsx

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import { Link } from "react-router-dom";
+
+function CardItem(props) {
```

Comment:
You were destructuring the props earlier and now you're not. It's better practice to destructure them, so this would be:

`function CardItem({ id, path, alt, text })`